### DISCUSSION TITLE
Finetuning with very low data (50-100 sentences)

### DISCUSSION THREAD

**Discussion by tcourat** [urchade/GLiNER#79] [General]
Has anyone tried to finetune Gliner on a task with a very low amount of data (like 50 sentences) ?

**Comment by wjbmattingly** [comment]
I have! It works quite well, depending on the complexity of the task.

**Reply by andrea-mar** [reply]
What metric did you use to select the best model during fine-tuning?
